Altered vs_redist text to be more visible

SirMangler opened the pull request:
This isn't as blatantly in your face as my previous PR, but there's a significant difference. This doesn't distract from the explanation of the development versions, and the position shows direct relevance to the downloads itself.

I feel this is satisfactory for all parties.

![pr](https://i.imgur.com/nsJLv1O.png)

SirMangler committed:
Altered vs_redist text to be more visible

AlexApps99 commented:
Maybe after downloading it could redirect to a page instructing how to extract/install VC++redist and the Dolphin archive, so absolutely nobody would be able to ignore/not notice it

MrMendelli commented:
> 
> 
> Maybe after downloading it could redirect to a page instructing how to extract/install VC++redist and the Dolphin archive, so absolutely nobody would be able to ignore/not notice it

Perhaps it would also be ideal to check if it is installed first as to avoid looking like nagware. Similar to software that takes you straight to the _"buy now"_ page upon each installation. That also begs the question of users that update from any version _prior_ to the new minimum requirement: Should this be checked for during the update process or is it negligible enough to only implement on the official site?

memequester commented:
Alternatively, the development version info box could use the same class as the stable version info box. Just requires a class change (`alert` to `alert alert-info`). Draws more attention to the information that people should be reading in the first place and it's more consistent with the rest of the page styling.

<img width="1164" alt="Screenshot 2020-04-22 at 02 20 47" src="https://user-images.githubusercontent.com/61253308/79930639-f3b94480-8440-11ea-9cd0-9990adb22f7c.png">
<img width="1165" alt="Screenshot 2020-04-22 at 02 22 11" src="https://user-images.githubusercontent.com/61253308/79930645-f61b9e80-8440-11ea-81f9-dd32e43ee944.png">

SirMangler commented:
> 
> 
> Alternatively, the development version info box could use the same class as the stable version info box. Just requires a class change (`alert` to `alert alert-info`). Draws more attention to the information that people should be reading in the first place and it's more consistent with the rest of the page styling.

While this is aesthetically pleasing, and I generally agree with this change, in my opinion it does not resolve the issue this PR was made to resolve, which is making the last line (referring to the C++ redistributable requirement) easier to see, and to draw more attention to it. I like the idea of having this line separate from the paragraph. 

**Idea one** is to isolate the line in the alert box. 
![image](https://user-images.githubusercontent.com/4237834/79972385-c8107b80-848d-11ea-96f8-f8629552b207.png)

**Idea two** is to take your suggestion and add whitespace, although I believe this makes it too chonky. 
![image](https://user-images.githubusercontent.com/4237834/79972634-2e959980-848e-11ea-91b1-153b03aae93f.png)

**Idea three**, the inverse of the first one:
![image](https://user-images.githubusercontent.com/4237834/79972784-61d82880-848e-11ea-8c5a-332f62ea11bb.png)


I personally lean towards idea three. Any opinions and input is much appreciated.

AlexApps99 commented:
I agree with memequester that the information about development versions should be in the same class of info box as the stable versions, for consistency's sake.

Other than that, I think that the redist text should be in a separate `alert alert-warning` info box above or below the developer version description. It would look like this:
![image](https://user-images.githubusercontent.com/30854646/79974837-16edfd80-84ee-11ea-8227-a648d6053881.png)

The bright color isn't too intrusive but it catches the eye of users installing the builds. This is important because from my personal experience (and SirMangler's too) we spend too much of our time in the [unofficial Dolphin Emulator Discord server](https://discordapp.com/invite/sRDuAEy) reminding users to install the redistributables while helping them update from their outdated stable builds.

SirMangler commented:
> Other than that, I think that the redist text should be in a separate `alert alert-warning` info box above or below the developer version description. It would look like this:
> ![image](https://user-images.githubusercontent.com/30854646/79974837-16edfd80-84ee-11ea-8227-a648d6053881.png)
>

I'm very fond of this.

memequester commented:
The colour change helps, but a few users on the unofficial Discord have noted the... unappealing colour.

Bootstrap class `alert alert-danger` could be used in place of `alert alert-warning`. I believe this is even more attention-grabbing while still fitting with the overall style

<img width="1368" alt="Screenshot 2020-04-22 at 12 12 37" src="https://user-images.githubusercontent.com/61253308/79975745-605d2f00-8493-11ea-8c98-7594977bbdea.png">
<img width="1369" alt="Screenshot 2020-04-22 at 12 17 55" src="https://user-images.githubusercontent.com/61253308/79975757-6521e300-8493-11ea-9b46-16318244518a.png">

I personally prefer the `alert` and `alert alert-danger`, as I think it looks cleaner, but `alert alert-info` and `alert alert-danger` is more consistent with the rest of the page's styling.

MayImilae commented:
IMO, I like memequester's approach (specifically the lower one). That's pretty good.

I said in the last PR that I don't believe this change is necessary, as the influx of reports of missing MSVC dlls would stop fairly quickly. That has happened, user questions about it have already faded away (on our forums at least). However, if this is done in a clean way that fits the aesthetic of the site and doesn't compromise functionality and needed information conveyance... then making the requirement more obvious is a good thing and we should probably do it. I think memequester's version (again, the lower one with a blue bar above a red bar) fulfills that the best. It keeps the description of what the dev builds are directly beneath the header for dev builds, as it should be, but then has a in-style alert for the needed MSVC version beneath it, one that is still obvious without intruding on core functionality and information. Works for me!

SirMangler committed:
Stylised development build information and warnings

Changed to be consistent with the rest of the webpage and website, as well as to make the vital information very clear.

SirMangler commented:
I absolutely agree with what was said, and I like memequester's stylisation. This perfectly meets the point of this PR and brings extra improvement. 

We still frequently receive reports of the error on our Discord, as was mentioned by AlexApps. Hopefully this should be the final nail on the coffin.

Is any extra changes required for the translation blocks to work as intended?

delroth commented:
I made it a bit more visible on https://dolphin-emu.org/download/#download-beta -- hopefully that will help.

SirMangler commented:
> 
> 
> I made it a bit more visible on https://dolphin-emu.org/download/#download-beta -- hopefully that will help.

Visually this is much worse in my opinion. The text should be below the alertbox with padding, not sitting directly ontop. There's also a really large unnecessary gap between the alertbox and the downloads page. Please compare with the stables section of the page, it visually is inconsistent. 

I've opened a new PR adding the changes from this PR.
https://github.com/dolphin-emu/www/pull/117
